getting_started/administrators.adoc

```diff
@@ -39,6 +39,12 @@ Docker and {product-title} must run on the Linux operating system. If you wish t
 run the server from a Windows or Mac OS X host, you should download and run the
 Origin Vagrant image as described in xref:building-from-source[Method 3].
 
+[CAUTION]
+====
+{product-title} and Docker use iptables to manage networking. Be careful
+that firewall and other software, when used, does not alter the setup.
```

Comment:
firewall isn't software...  Perhaps "local firewall rules, or other software that makes changes to iptables"